kiwi_keg/tools/compose_kiwi_description.py

```diff
@@ -86,21 +88,32 @@
         converted if necessary. Conversion from 'osc' is not supported.
         [default: json]
 
+    --purge-stale-files=<true|false>
```

Comment:
could we use the presence of the flag as true and the lack of presence/omission as false instead of setting a value ?
i.e.
`cmd ... --purse-stale-files` -> True, purge

Reply:
If the default behaviour is to purge, maybe same approach of present or not present but different name
`--not-purge-stale-files` if not present, files would be purged